fdm-data/src/cultivations/index.ts

```diff
@@ -1,65 +1,32 @@
-import {
-    type FdmType,
-    addCultivationToCatalogue,
-    type fdmSchema,
-    getCultivationsFromCatalogue,
-} from "@svenvw/fdm-core"
 import { getCatalogueBrp } from "./catalogues/brp"
+import type { CatalogueCultivation, CatalogueCultivationName } from "./d"
 
 /**
- * Extends the cultivations catalogue in the Farm Data Model (FDM) with data from a specified source.
+ * Retrieves a cultivation catalogue based on the specified name.
  *
- * @param fdm - An instance of the FDM, providing database access.
- * @param catalogueName - The name of the catalogue to extend. Currently, only 'brp' is supported.
+ * This function acts as a dispatcher, selecting and returning the appropriate
+ * cultivation catalogue based on the provided `catalogueName`.
  *
- * @throws An error if the specified catalogue name is not recognized.
+ * @param catalogueName - The name of the desired cultivation catalogue.
+ *                        Currently supported names are: "brp".
+ * @returns An array of `CatalogueCultivationItem` objects representing the
```

Comment:
brp is here then a catalogue for the NL i suppose? do you expect country specific catalogues all with their own name or prescripe already potential names such as b_lu_nl rather than brp?

Reply:
Yes I expect list of cultivations from other counties, but it can also be a custom list of crop varieties for a specific producer. As long as you can provide the properties it does not matter

Reply:
max flexibility... :-)